lib/api/entries/index.js

```diff
@@ -96,8 +96,22 @@ function configure (app, wares, ctx) {
       // such as enforcing a type property to exist, are followed.
       return res.format({
         text: function ( ) {
+          res.set("Content-Type", 'text/plain');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2165370020/issues/source?bid=2452379&fileBranchId=3160529#l99)